python/csrc/batch_prefill.cu

```diff
@@ -76,10 +76,13 @@ std::vector<torch::Tensor> BatchPrefillWithPagedKVCachePyTorchWrapper::Run(
   CHECK_INPUT(q);
   CHECK_INPUT(qo_indptr);
   if (paged_kv_defined) {
-    CHECK_INPUT(paged_kv_cache.value());
+    CHECK_CUDA(paged_kv_cache.value());
+    CHECK_LAST_DIM_CONTIGUOUS(paged_kv_cache.value());
```

Comment:
Maybe we need a new macro here, like `CHECK_INPUT_LAST_DIM_CONTIGUOUS()`?

@yzh119